source/sass/print.scss

```diff
@@ -0,0 +1,58 @@
+@media print {
+  @page {
+    margin: 2cm;
+  }
+
+  body {
+    color: $black;
+    background: $white;
+    width: 100%;
+  }
+
+  video {
+    display: none;
+  }
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6,
+  ul,
+  ol,
+  img {
+    page-break-after: avoid;
+  }
+
+  .container {
+    width: auto;
+
+    .row {
+      @extend .d-print-block;
+    }
+  }
+
+  a {
+    &:not(.btn) {
+      text-decoration: none;
+    }
+
+    &[href]:after {
+      content: " (" attr(href) ")";
```

Comment:
ooh, I like this, very smart.

Reply:
Include URL of a hyperlink after the link text
For example: 
On a website, we show [I AM A HYPERLINK](https://example.com).
On print mode, we show I AM A HYPERLINK(https://example.com)